### PR TITLE
Add gobstones inputs - Fixes #124

### LINKED ISSUE #124: Support gobstones attires input
We need a way to: 

* upload the attires
* indicate that the exercise will use it 
   * :up: this seems to have being solved with `<gs-attire>`

### PR DESCRIPTION
**Pull request by rodri042**
Fixes #124 

Demo: https://www.youtube.com/watch?v=PvHRxcpQ4pk

This depends on https://github.com/mumuki/mumuki-bibliotheca-api/pull/116

### CODE REVIEW

**Comment by flbulgarelli** [comment]
@fedescarpa I have already discussed with @rodri042 about reaming and reusing the `/images` endpoint - as `/assets`. 

Aside of that, from a functional point of view this PR is OK. But I need your feedback about visuals and angular code.

File: src/scripts/controllers/create-gobstones-toolbox-controller.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  <block type="Poner"></block>
+</category>
+<category name="Procedimientos primitivos">
+</category>
+<category name="Funciones primitivas">
+</category>
+<category name="Mis procedimientos" custom="PROCEDURE_CALLS">
+</category>
+<category name="Mis funciones" custom="FUNCTION_CALLS">
+</category>
```
Comment (fedescarpa) [inline src/scripts/controllers/create-gobstones-toolbox-controller.js]: Localize these categories names

File: src/scripts/controllers/create-gobstones-attire-controller.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+          !rule.image || _.some(
+            COLORS,
+            (color) => {
+              const value = rule.when[color];
+              return !_.isFinite(parseInt(value)) && value !== "*" && value !== "+";
+            }
+          )
+        ) || _.some($scope.borders, (border) => $scope.attire.borders[border] === null)
+      );
+    };
```
Comment (fedescarpa) [inline src/scripts/controllers/create-gobstones-attire-controller.js]: too much logic. Please delegate it

**Comment by fedescarpa** [comment]
I really like it. fix my comments and... Is possible to create xml and json files into assets folders in the repo. I dont like to use a gist.

File: src/scripts/controllers/create-gobstones-toolbox-controller.js (hunk 1 above)
Comment (rodri042) [reply src/scripts/controllers/create-gobstones-toolbox-controller.js]: Done, excepting the categories `Procedimientos primitivos` and `Funciones primitivas` that need to be called that way in order to work

**Comment by rodri042** [comment]
@fedescarpa please review again. Now it uploads the images and JSON to the repository using the `/assets` endpoint

**Comment by fedescarpa** [comment]
@rodri042 assets endpoint fails and I, don't know why. Maybe is otrher endpoint, like images, it would be nice to rename it to assets and test the upload one more time.

Otra posibilidad es que tenga mucho sueño y no encuento por que me falla al probarlo localmente...

**Comment by flbulgarelli** [comment]
> assets endpoint fails and I don't know why. Maybe is other endpoint, like images. It would be nice to rename it to assets and test the upload one more time.

@fedescarpa are you using latest bibliotheca-api?

**Comment by fedescarpa** [comment]
No, I'm using an old one, hahahaha

**Pull request merged**